c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -2277,13 +2277,6 @@ void MfxC2DecoderComponent::WaitWork(MfxC2FrameOut&& frame_out, mfxSyncPoint syn
     MFX_DEBUG_TRACE_I32(m_mfxVideoParams.mfx.FrameInfo.CropW);
     MFX_DEBUG_TRACE_I32(m_mfxVideoParams.mfx.FrameInfo.CropH);
 
-#if MFX_DEBUG_DUMP_FRAME == MFX_DEBUG_YES
-    static int frameIndex = 0;
-    uint8_t stride = frame_out.GetC2GraphicView()->layout().planes[C2PlanarLayout::PLANE_Y].rowInc;
-    static YUVWriter writer("/data/local/tmp",std::vector<std::string>({}),"decoder_frame.log");
-    writer.Write(mfx_surface->Data.Y, stride, frame_out.GetC2GraphicBlock()->height(), frameIndex++);
-#endif
-
```

Comment:
Will this logic still work? I remember Yichi added this code.

Reply:
No, confirmed by Yichi. These codes did not work.